packages/@o3r/schematics/src/rule-factories/ng-add/index.ts

```diff
@@ -16,6 +17,10 @@ import { getExternalDependenciesVersionRange, getNodeDependencyList, getPackageM
  * @param packageJsonPath path of the package json of the project where they will be installed
  */
 export function ngAddPackages(packages: string[], options?: NgAddPackageOptions, packageJsonPath = '/package.json'): Rule {
+  if (options?.workingDirectory && !packageJsonPath.startsWith(options.workingDirectory)) {
```

Comment:
I suppose that this is to convert relative path to absolute on, the `startWith` may not be the best on.
```typescript
if (options?.workingDirectory && !path.isAbsolute(options.workingDirectory)) {
```
?

Reply:
Not really. It's to add the working directory at the beginning of the **packageJsonPath** if it's not already the there.